docs/guides/styled-components.md

```diff
@@ -0,0 +1,116 @@
+# Styled components
+
+Ah yes, begone are the old days of css files, getting a scss or sass compiler working,... Welcome styled components!
+The benefits are:
+
+- Dynamic css, pass js variables to your css.
+- Unique classnames, no magic dangling css class that mess up everything.
+
+## There's a plugin for that
+
+Get the plugin and install it according to it's [documentation](https://github.com/nozzle/react-static/tree/master/packages/react-static-plugin-styled-components).
```

Comment:
It's such a minimal documentation that it might make sense to add the 2 steps here.

Reply:
I didn't because it might change and then we need to update it in 2 places